src/app/core/entity-components/entity-subrecord/entity-subrecord/value-accessor.ts

```diff
@@ -8,17 +8,17 @@
 export function getReadableValue<OBJECT, PROPERTY extends keyof OBJECT>(
   data: OBJECT,
   key: PROPERTY
-): OBJECT[PROPERTY] | string {
-  if (isConfigurableEnum(data, key)) {
-    return (data[key] as any).label;
+): any {
+  const value = data[key];
+  if (isConfigurableEnum(value)) {
+    return (value as any).label;
+  } else if (Array.isArray(value)) {
+    return (value as Array<any>).map((v) => getReadableValue({ v }, "v"));
```

Comment:
```suggestion
    return (value as Array<any>).map((v) => getReadableValue({ v }, "v"));
     return value.map((v) => getReadableValue({ v }, "v"));
```

This should already be type-asserted